appveyor.yml

```diff
@@ -21,25 +21,28 @@ matrix:
       MSYSTEM: MSVC
 
 for:
-  -
-    matrix:
+  - matrix:
       only:
         - MSYSTEM: MINGW64
+    install:
```

Comment:
Any advantage of splitting it up?

Reply:
AppVeyor provides (slightly) better error output when prerequisite installations are conducted in the `install` section.

It also matches the way I handle appveyor.yaml in my fork of Ninja, which has several other CI targets on appveyor, which all install re2c (I remove the generated code that is stored in git).

Reply:
Supplemental information 

The `install` section for the MINGW target looks identical to how it's done in the `build_script` section, because it is.

But AppVeyor has special handling for the other package managers that they support, when used in the `install` section.

That's not obvious in this PR because the Ninja project is only using the downloaded re2c package on the MINGW target (relying on the bundled pre-generated code for the others), but using the `install` section is significantly nicer looking for the other AppVeyor platform types.